Development.podspec

```diff
@@ -61,6 +61,10 @@ Pod::Spec.new do |s|
 
   s.test_spec 'TicTacToeTests' do |test_spec|
     test_spec.dependency 'Development/SampleTicTacToe'
+    test_spec.dependency 'WorkflowTesting'
+    test_spec.dependency 'BackStackContainer'
```

Comment:
I needed to add these as not including them would result in a 
`ld: warning: Auto-Linking framework not found BackStackContainer Undefined symbols for architecture x86_64`: